website/static/js/filepage/index.js

```diff
@@ -17,6 +18,65 @@ var Panel = utils.Panel;
 
 var EDITORS = {'text': FileEditor};
 
+var MakeClient = function(element) {
```

Comment:
However, you will want to to use the second argument to the config function, `isInitialized` to ensure that this is only called once when the copy button is mounted.